src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1029,6 +1029,12 @@ def forward(
         hidden_states = inputs_embeds + positions
         hidden_states = nn.functional.dropout(hidden_states, p=self.dropout, training=self.training)
 
+        if self.gradient_checkpointing and self.training:
+            if use_cache:
+                logger.warning(
```

Comment:
```suggestion
                logger.warning(
```
This should be `warn_once` here (or `warning_once`) can't rememver the syntax.

Reply:
@gante done !